content/en/flux/integrations/gcp.md

```diff
@@ -109,16 +114,16 @@ serviceAccount:SA_NAME@PROJECT_ID.iam.gserviceaccount.com
 
 #### To Kubernetes Service Accounts
 
-The principal string for a Kubernetes Service Account from a *GKE cluster* has the
+The IAM Principal for a Kubernetes Service Account from a *GKE cluster* has the
 following format:
 
 ```
-principal://iam.googleapis.com/projects/PROJECT_NUMBER/locations/global/workloadIdentityPools/PROJECT_ID.svc.id.goog/subject/ns/NAMESPACE/sa/KSA_NAME
+serviceAccount:PROJECT_ID.svc.id.goog[NAMESPACE/KSA_NAME]
```

Comment:
This is a well-known/documented alternate form. It's the only one that works for the GKE integration.